jkube-kit/resource/service/src/test/java/org/eclipse/jkube/kit/resource/service/WriteUtilTest.java

```diff
@@ -120,7 +133,7 @@ void writeResourcesIndividualAndComposite_withResourcesWithSameName_shouldWriteA
     // When
     WriteUtil.writeResourcesIndividualAndComposite(klb.build(), resourceFileBase, null, log);
     // Then
-    verifyResourceUtilSave(null, 6);
+    //verifyResourceUtilSave(null, 6);
```

Comment:
I checked on master branch. Even if I comment this test line on master branch all tests pass. While setting a breakpoint in WriteUtil.java:91 I can see that `resourceFileBase` is never passed as `null`.

Reply:
Great, Thanks Rohan on confirming